extensions/interactions/baseValidatorSpec.js

```diff
@@ -30,12 +30,12 @@ describe('Interaction validator', function() {
   var currentState, otherState, goodOutcomeDest, goodOutcomeFeedback;
   var badOutcome, goodAnswerGroups, goodDefaultOutcome;
 
-  var createAnswerGroup = function(outcome, ruleSpecs, correct) {
-    if (!ruleSpecs) {
-      ruleSpecs = [];
+  var createAnswerGroup = function(outcome, rules, correct) {
+    if (!rules) {
+      rules = [];
     }
     return {
-      ruleSpecs: ruleSpecs,
+      rules: rules,
       outcome: outcome,
       correct: !!correct
     };
```

Comment:
Was looking at this and am thinking it should use the factory, though the !!correct baffles me so wasn't sure if I should touch it